packages/jest-runtime/src/__tests__/script_transformer.test.js

```diff
@@ -509,4 +509,23 @@ describe('ScriptTransformer', () => {
     expect(fs.readFileSync).not.toBeCalledWith(cachePath, 'utf8');
     expect(writeFileAtomic.sync).toBeCalled();
   });
+
+  it('does not reuse the in-memory cache between different projects', () => {
+    const scriptTransformer = new ScriptTransformer({
+      ...config,
```

Comment:
I don't think this is safe for older versions of node unfortunately

Reply:
Ohh, I thought `jest` tests were also transformed 😅 I'm gonna use `Object.assign` then

Reply:
Yeah I forget this every time 😸

Reply:
Btw, this works now that we've migrated to babel 7 - as part of that we went with `@babel/preset-env` instead of manually listing the transforms